packages/editor-ui/src/components/RunDataSchemaItem.vue

```diff
@@ -38,9 +40,14 @@ const key = computed((): string | undefined => {
 const schemaName = computed(() =>
 	isSchemaParentTypeArray.value ? `${props.schema.type}[${props.schema.key}]` : props.schema.key,
 );
-const text = computed(() =>
-	Array.isArray(props.schema.value) ? '' : shorten(props.schema.value, 600, 0),
-);
+const text = computed(() => {
+	if (Array.isArray(props.schema.value)) return '';
+
+	return sanitizeHtml(shorten(props.schema.value, 600, 0)).replaceAll(
+		'\n',
+		`<span class="${$style.newLine}">\\n</span>`,
```

Comment:
You could avoid having to use innerHTML by splitting on `\n` instead and then inserting the newLine span in a v-for

Reply:
That's a great idea! I updated the code, can you please re-review? 🙏